section3/reflection.md

```diff
@@ -2,12 +2,23 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+That didn't feel good at first at all.  Once I started getting the hang of things a little more, the challenges started to get more fun.  
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+I use youtube alot where they probably use arrays when definitely youtube playlists for certain videos in a category.
 
 3. In your own words, explain what an Array method is.
 
+An array method is having a bunch of variables under a single variable name.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+The purpose of a loop is if I want my program to perform an action a specific number of times.  The loop allows me to write the code for that action one time instead of multiple lines of code.  I would use a loop in conjunction with an array when i'm dealing with many variables that I want to carry out an action on each one.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+In stead of telling him to clean his room every day multiple times. I just write it on the big white board hanging on his wall, "Is your room clean?" one time.
+
 6. What questions do you still have about Arrays and loops?
+
+I saw the variable declarations const and let.  Why am I to read them as var?
```

Comment:
`var` is the 'OG' approach to variable declaration in JS. `let` and `const` are more modern ways to define variables, which ultimately convert to `var` under the hood. You'll learn more about the modern ES6 syntax in Mod 2 and beyond once you have a foundation in the basics.